test/e2e/atlas/operator_helper_test.go

```diff
@@ -167,7 +158,7 @@ func (oh *operatorHelper) stopOperator() {
 		oh.t.Errorf("unable to retrieve operator deployment: %v", err)
 	}
 
-	deployment.Spec.Replicas = toptr.MakePtr(int32(0))
+	deployment.Spec.Replicas = akov2toptr.MakePtr(int32(0))
```

Comment:
we have our own that I'd like to use instead of more dependencies with kube
```suggestion
	deployment.Spec.Replicas = pointer.Get(0)
```